src/main/java/az/keytd/expensetracker/controller/ExpenseController.java

```diff
@@ -5,17 +5,17 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import az.keytd.expensetracker.entities.Expenses;
+import az.keytd.expensetracker.entities.Expense;
 import az.keytd.expensetracker.service.ExpensesService;
 
 @RestController
 @RequestMapping("Expenses")
-public class ExpensesController {
+public class ExpenseController {
     @Autowired
     private ExpensesService expensesService;
 
     @GetMapping("get/accountId")
```

Comment:
no need to mention "get/" http method "GET" already tells as what this API is gonna do. rename path to "byAccountId"